src/lib/get_cmdline.c

```diff
@@ -741,7 +741,7 @@ int libreport_get_mountinfo_for_mount_point(FILE *fin, struct mountinfo *mntnf,
         }
 
         size_t len = (pos_cur - pos_bck);
-        mntnf->mntnf_items[fn] = g_malloc(sizeof(char) * (len));
+        mntnf->mntnf_items[fn] = g_malloc(sizeof(char) * (len + 1));
```

Comment:
Perhaps `g_malloc(len + 1)` would clearer. `sizeof(char)` is always 1.

Reply:
In this case it would be a bit awkward, but, generally, something like `sizeof (*(mntnf->mntnf_items[fn]))` would be better IMO.

Reply:
See latest commit.